lang/cs/Org.Apache.REEF.IO.Tests/TestHadoopFileSystem.cs

```diff
@@ -142,6 +143,20 @@ public void TestCreate()
             Assert.Throws<NotImplementedException>(() => _fileSystem.Create(GetTempUri()));
         }
 
+        [Fact(Skip = "These tests need to be run in an environment with HDFS installed.")]
+        public void CreateUriForPathNoPrefix()
+        {
+            var uri = _fileSystem.CreateUriForPath("/tmp/TestHadoop");
+            Assert.True(new Regex("hdfs://[a-z]+:\\d+/tmp/TestHadoop").Match(uri.AbsoluteUri).Success);
+        }
+
+        [Fact(Skip = "These tests need to be run in an environment with HDFS installed.")]
+        public void CreateUriForPathWithPrefix()
+        {
+            var uriString = "hdfs://localhost:9000/tmp/TestHadoop";
+            Assert.Equal(uriString, _fileSystem.CreateUriForPath(uriString).AbsoluteUri);
```

Comment:
Can we do this without calling "AbsoluteUri"? Seems like possiblity of unintended functionality.

```csharp
var uri = new Uri("hdfs://localhost:9000/tmp/TestHadoop");
Assert.Equal(uri, _fileSystem.CreateUriForPath(uri.ToString()));
```

Reply:
Done.